help/submit_tex.md

```diff
@@ -142,7 +142,7 @@ TeX-based supplemntal material should be included in the main document root dire
   ./supplement.bbl
 ```
   

```

Comment:
Wasn't part of the original diff, but by chance I wanted a bit more context and opened the area above this diff. There's a typo on line 135, `supplemental`.

Reply:
Fixed (and another minor typo I noticed)!